kiva/agg/src/kiva_graphics_context.h

```diff
@@ -588,7 +588,7 @@ namespace kiva
 
 
             agg24::path_storage img_outline = img.boundary_path(img_mtx);
-            agg24::rendering_buffer* src_buf = img.rendering_buffer_ptr();
```

Comment:
Right. So everywhere you used `agg_pixfmt` in this method, it appears you should have used `other_format` instead. Did that not compile for you?

I've tested this locally (macOS) and everything appears to be working as it always did.